New dbt cloud IA (#2932)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/about-access.md

## FAQs

> - **When are IdP group memberships updated for SSO Mapped groups?** Group memberships

Author: ```suggestion
- **When are IdP group memberships updated for SSO Mapped groups?**  <br />
```

>   are updated every time a user logs into dbt Cloud via a supported SSO provider. If

Author: Group memberships are updated whenever a user logs into dbt Cloud via a supported SSO provider. If you've changed group memberships in your identity provider or dbt Cloud, ask your users to log back into dbt Cloud to synchronize these group memberships.

>   are updated every time a user logs into dbt Cloud via a supported SSO provider. If

Author: ```suggestion
 Group memberships are updated whenever a user logs into dbt Cloud via a supported SSO provider. If you've changed group memberships in your identity provider or dbt Cloud, ask your users to log back into dbt Cloud to synchronize these group memberships.
```

>   you've changed group memberships in your identity provider or dbt Cloud, ask your

Author: ```suggestion

```

>   users to log back into dbt Cloud for these group memberships to be synchronized.

Author: **Can I set up SSO without RBAC?**  <br />

>   users to log back into dbt Cloud for these group memberships to be synchronized.

Author: ```suggestion
- **Can I set up SSO without RBAC?**  <br />
```

> 

Author: ```suggestion
Yes, see the documentation on [Manual Assignment](#manual-assignment) above for more information on using SSO without RBAC.
```

> - **Can I set up SSO without RBAC?** Yes, see the documentation on

Author: ```suggestion

```

>   [Manual Assignment](#manual-assignment) above for more information on using

Author: ```suggestion
- **Can I configure a user's License Type based on IdP Attributes?** <br />
```

>   SSO without RBAC.

Author: ```suggestion
  Yes, see the docs on [managing license types](/cloud-seats-and-users#managing-license-types) for more information.
```

> - **Can I configure a user's License Type based on IdP Attributes?** Yes, see

Author: ```suggestion
- **Why can't I edit a user's group membership?**  <br />
```

>   the docs on [managing license types](/cloud-seats-and-users#managing-license-types)

Author: ```suggestion
Make sure you're not trying to edit your own user as this isn't allowed for security reasons. To edit the group membership of your own user, you'll need a different user to make those changes.
```

>   for more information.

Author: ```suggestion

```
